bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/FindReplaceMessages.java

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Vector Informatik GmbH and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Vector Informatik GmbH - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.ui.internal.findandreplace;
+
+import org.eclipse.osgi.util.NLS;
+
+final public class FindReplaceMessages extends NLS {
+
+	public static final String BUNDLE_NAME = FindReplaceMessages.class.getName();
+
+	private FindReplaceMessages() {
+		// Do not instantiate
+	}
+
+	static {
+		NLS.initializeMessages(BUNDLE_NAME, FindReplaceMessages.class);
+	}
+
+	public static String FindReplace_Status_replacement_label;
+	public static String FindReplace_Status_replacements_label;
+	public static String FindReplace_Status_noMatchWithValue_label;
+	public static String FindReplace_Status_wrapped_label;
+	public static String FindReplaceDialog_read_only;
+	public static String FindReplace_Status_selections_label;
+	public static String FindReplace_Status_selection_label;
+	public static String FindReplace_Status_noMatch_Label;
```

Comment:
There is no label defined with this ID. Seems to be a duplicate with a typo `FindReplace_Status_noMatch_Label` -> `FindReplace_Status_noMatch_label`.
Leads to runtime warnings:
```
Warning: NLS missing message: FindReplace_Status_noMatch_Label in: org.eclipse.ui.internal.findandreplace.FindReplaceMessages
```

Reply:
I have removed all labels that weren't in use and - in particular - removed labels which differed only in case.

https://github.com/eclipse-platform/eclipse.platform.ui/commit/555a45e5a86580594a0b35108879fa33b98de879